From ca7e15bd615605ea5d4ed39825cd4532399f3daf Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <josef.schlehofer@nic.cz>
Date: Tue, 27 Aug 2019 14:16:43 +0200
Subject: [PATCH] Revert "knot: fix lmdb dependency"

This reverts commit 52fcf837bc32fe4a388733ba3d2969e01c6d6153.
As lmdb is not present in OpenWrt 18.06.
---
 net/knot/Makefile | 2 +-
 1 file changed, 1 insertions(+), 1 deletions(-)

diff --git a/net/knot/Makefile b/net/knot/Makefile
index 92267c77f..1a00119cc 100644
--- a/net/knot/Makefile
+++ b/net/knot/Makefile
@@ -44,7 +44,7 @@ endef
 define Package/knot-libs
 	$(call Package/knot-lib/Default)
 	TITLE+= common DNS and DNSSEC libraries
-	DEPENDS+=+libgnutls +lmdb
+	DEPENDS+=+libgnutls
 endef

 define Package/knot-libzscanner
--
2.20.1
